propolis/src/hw/nvme/mod.rs

```diff
@@ -422,16 +424,14 @@ pub struct PciNvme {
 impl PciNvme {
     /// Create a new pci-nvme device with the given values
     pub fn create(
-        vendor: u16,
-        device: u16,
```

Comment:
Why not keep this stuff as a parameter?

Reply:
NVMe was the only device that took them as parameters so I was just matching everywhere else.